src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/WebViewTests/UnoSamples_Test_NavigateToString.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.WebViewTests
+{
+	[TestFixture]
+	public partial class WebView_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void WebView_NavigateToLongString()
```

Comment:
The test is failing on WASM: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=9289&view=ms.vss-test-web.build-test-results-tab

This isn't unexpected since `WebView` isn't implemented yet for WASM. You can ignore it on WASM using the following attribute: `[ActivePlatforms(Platform.Android, Platform.iOS)]`

Reply:
I asked on Lobby if I should restrict test to not be run on WASM, and get answer (if I remember correctly, from Jerome Laban) that I should do nothing to it...
So, add this attribute or leave it as it is?

Reply:
It depends if you want the PR to pass. :p

Reply:
I want, of course :)